targets/TARGET_TOSHIBA/TARGET_TMPM46B/device/TOOLCHAIN_ARM_STD/tmpm46bf10fg.sct

```diff
@@ -31,6 +39,11 @@
 #endif
 
 #define Stack_Size MBED_CONF_TARGET_BOOT_STACK_SIZE
+#define VECTOR_SIZE 0x320
+
+#define RAM_FIXED_SIZE (MBED_CONF_TARGET_BOOT_STACK_SIZE + VECTOR_SIZE)
+#define MBED_IRAM1_START (MBED_RAM_START + VECTOR_SIZE)
```

Comment:
Should `MBED_IRAM1_START` be equal to `0x20000000 + 0x1E0` as was the case in line 46?